pyramid_swagger/tween.py

```diff
@@ -18,6 +18,8 @@
 from bravado_core.response import get_response_spec
```

Comment:
Added it. I also double-checked the issue reported in #200 and indeed we don't read that setting correctly. I also updated our documentation, as most of the examples given there are actually wrong and don't work the way they're specified.